templates/actors/group/member.hbs

```diff
@@ -86,11 +86,17 @@
         {{else}}
         <div class="info">
             <div>
-                <i class="fa-solid fa-shield" data-tooltip aria-label="{{ localize "DND5E.ArmorClass" }}"></i>
+                <i class="fa-solid fa-shield"
```

Comment:
Right now this is a pretty small hover area, it would probably be nicer if this was on the containing `<div>` so that hovering over the icon or the value would show the tooltip. That would also prevent the tooltip from concealing the value when hovered.